python/jupyterlab-chat/jupyter-config/nb-config/jupyterlab_chat.json

```diff
@@ -0,0 +1,7 @@
+{
+  "NotebookApp": {
```

Comment:
Not sure we need this and the related change in `pyproject.toml`, since the `NotebookApp` config was for the now deprecated classic notebook server?

Reply:
Thanks @jtpio, I wasn't sure about that.